### DISCUSSION TITLE
I can't prove my academic status

### DISCUSSION THREAD

**Discussion by fdoganozcan** [community/community#19025] [GitHub Education] [answered]
Hi, there is no date on the student cards at the university I am studying at, and I have no idea how to prove it. I have a college card with my name on it and an active email address.

**Comment by melvincwng** [comment, marked as answer]
Hi @fdoganozcan, first of all, let me welcome you to GitHub Community :)

I assume you need to prove your academic status to apply for the GitHub Student Developer pack. If that's the case, continue reading on.

With regards to your application for GitHub Student Developer pack, you can consider submitting documents such as the following:

1. Picture of your school ID with current enrollment date
2. Academic transcript
3. School affiliation or enrollment verification letter
4. Class schedule

If all fails, please submit a ticket to Github Education Support over [here](
https://support.github.com/contact/education)

Hope this helps!

Reference:
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack

**Reply by melvincwng** [reply]
@fdoganozcan, if you find the answer useful, please kindly 'Mark as answer' so that others know this issue has been addressed.

Thank you 😊

**Reply by fdoganozcan** [reply]
> 

Thank you @melvincwng 🙏

**Reply by marksimi** [reply]
@melvincwng, this is unfortunately marked as a Helpful Answer...but Github no longer even does individual reviews on tickets.  It routes to a chatbot which does nothing.  Please see below:

![CleanShot 2024-02-20 at 15 12 18@2x](https://github.com/community/community/assets/3029809/9021644a-8774-4d6a-b3b7-a8e1f483dd08)

I've attempted submission probably 50 times now with different permutations of documents.  The model refuses to parse my last name, saying GH billing differs from my transcript.

Since there's no escalation path whatsoever; I'm effectively shut out of getting a Student Developer pack.  Which I've been trying to do for the past 13 months.

**Reply by Garry028** [reply]
I'm uploading my I'd card still it showing it should have college name 😔

**Comment by AIndoria** [comment]
Having a similar issue over here. Github won't accept even the 'accepted' forms of ID, like a dated bursar statement or a class schedule. Moreover, they disabled the "Upload" button and now I have to take a picture of the blurry document from their online website thing that doesn't help lol. The IDs and transcripts the college provides aren't dated either.

Sent a support ticket but hey.

**Reply by sharklayer** [reply]
No one here or in support will help you to solve the problem. I have 34 requests all instantly failed. Github support is awful...

**Reply by aisha-yaseen** [reply]
> Having a similar issue over here. Github won't accept even the 'accepted' forms of ID, like a dated bursar statement or a class schedule. Moreover, they disabled the "Upload" button and now I have to take a picture of the blurry document from their online website thing that doesn't help lol. The IDs and transcripts the college provides aren't dated either.
> 
> Sent a support ticket but hey.

That is exactly what I am facing right now, Wanna apply for a student developer pack, but can't upload my documents.

**Reply by jrh079** [reply]
Same problem here, have several tickets opened.  The ones pertaining to Student Enrollment Verification, auto close with no live help at all :(

**Reply by leventozz** [reply]
Same problem. Also back camera doesn't work on that page. It is impossible to prove that you are a student.

**Reply by alanluke2002** [reply]
also experiencing a similar problem, cannot upload a photo or access my computers camera to take a photo

**Reply by jf2024** [reply]
also same problem, uploaded an enrollment verification picture but somehow still denied??? like bruh fix this, cant be that hard

**Reply by paulserreku** [reply]
I have the same problem, how I can fixe it ?

**Reply by jf2024** [reply]
ok so i actually followed someone instructions below: 
Experience on Feb 2024: compress the jpg file with a size smaller than 500 KB. Otherwise, it won't process.
This is weird, because the page says that the limit size is 1 MB. Wasted me quite some time until I saw this thread

basically, whatever enrollment verification you got, just compress it less then 500kb, that solved it for me

**Reply by steb6** [reply]
@jf2024 thanks a lot for sharing this problem. It is absurd that there is no error message

**Reply by Birney-D** [reply]
I would do that but the only option is to use my camera to take a picture of the document.

**Comment by adubey14** [comment]
I am having a similar problem. is this just a way for github to not provide education support.

**Reply by AIndoria** [reply]
I had to create a manual support ticket and had to wait over a week for them to approve it. Try it, upload proof there. Maybe they will help.

**Reply by aisha-yaseen** [reply]
I did, Just right now, I hope I'll get approved soon

**Reply by guikaua12** [reply]
Hi, did you get approved?

> I did, Just right now, I hope I'll get approved soon

**Reply by marksimi** [reply]
This continues to be a terrible experience.

**Reply by jrh079** [reply]
Tried creating a ticket which they closed and said use portal or phone, on phone it only allows front facing camera you cant hit take photo button and aim correctly.  This whole process causing me to have panic attacks, its beyond difficult and I have worked desktop support for several decades.  Its clear to me they dont want this to be a simple verification otherwise you could upload or paste portal screenshots for verification.  Closing tickets automatically is also really poor customer service.

**Comment by aisha-yaseen** [comment]
Hey,
No not yet.

On Sat, 15 Oct 2022 at 20:49, Guilherme Kauã da Silva <
***@***.***> wrote:

> Hi, did you get approved?
>
> I did, Just right now, I hope I'll get approved soon
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/19025#discussioncomment-3885913>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMDUV62CUFCCNC2AZHCVU3TWDLG7RANCNFSM5ZLJSWXQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
[image: photograph]
*Aisha Khan*
Cloud Native Computing Developer
email: ***@***.***
phone: +92 316-6267820
Saylani, Karachi Pakistan
                                       LinkedIn
<https://www.linkedin.com/in/aisha-khan-6842a91a3/> | Github
<https://www.github.com/aishakhan0925> | Twitter
<https://www.twitter.com/aishakhan0925>

**Comment by ronaldo-mckenzie** [comment]
I did the same, and still no reply.

**Comment by MidKnightXI** [comment]
same on my side what a shame ...

**Comment by Spawin** [comment]
Same...

**Comment by crypt096** [comment]
Same here...

**Comment by SandiyosDev** [comment]
Exactly the same here, I wanted and needed educational help so that I can start contributing to open-source projects on GitHub... Why? Because the sense of accomplishment when getting your pulls merged is the greatest thing ever, and I want to level up my ability through Copilot to see how far I can get... But I guess not, customer support doesn't even provide verification when the verification portal itself fails miserably, there's just a sense of dissatisfaction and rage like I never had before, f*** GitHub and their lax options and stupid portal, I'm done contributing.

**Comment by sv2pddf** [comment]
Facing the same problem as of christmas 2023. uploaded a jpeg of an official pdf i can request from my university and even created the account with my .edu email. and YES the picture is neither too small nor too large.
upload of proof is bug-fest smh

**Reply by Fionayjx** [reply]
how did you upload the thing, It seems like only using a web camera?

**Reply by tauseef24** [reply]
I'm able to upload. But after clicking process my application, it's infinitely stuck as processing information.
![Screenshot 2024-01-10 225220](https://github.com/community/community/assets/73964188/7fa7f9ab-4019-4e3a-8967-118d00a387a0)

**Reply by H0r1z0ntal** [reply]
@tauseef24 , I had the same issue with my son and we managed to fix it by compressing the `.jpg` image with an online image compressor. I was able to upload a `.jpg` with the size of 500 KB and it got accepted.

**Reply by Naaberle** [reply]
How did you upload the jpg? I only have the option for webcam, can't actually upload any file.

**Reply by syrte** [reply]
Experience on Feb 2024: compress the jpg file with a size **smaller than 500 KB**. Otherwise, it won't process. 
This is weird, because the page says that the limit size is 1 MB. Wasted me quite some time until I saw this thread.

**Reply by Kacem300** [reply]
Thank you it works !!

**Comment by ghost** [comment]
This is so annoying... I honestly want this developer pack for students and my school is a Montessori one, and they don't give us a lot of options for proofs

**Reply by ghost** [reply]
I need help! PLEASE

**Comment by Kalondepatrick** [comment]
Same issue here.

**Reply by samus-aran** [reply]
Hi @Kalondepatrick 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I have a double-sided Student ID with the date on the other side. How do I submit this as evidence?](https://github.com/orgs/community/discussions/111352#double-sided-student-id-with-the-date-on-the-other-side)

**Reply by cagix** [reply]
> Hi @Kalondepatrick
> 
> We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

@samus-aran Thanks for the links. Unfortunately, this does not help: The current website does only offer to take a live photo using the front camera. There is simply no chance to submit a high quality image or document or even a scan :/

**Comment by jvanderhyde** [comment]
I have the same problem of the "Upload proof" page reappearing with no error banner (as a teacher). There seems to be a secret maximum of 500 MB. I increased the JPEG compression until I was just under 500 MB, and it worked. There also seems to be a secret minimum resolution, because my file was above the stated 1024x768 resolution, but it was still rejected with "resolution too low."

**Reply by samus-aran** [reply]
Hi @jvanderhyde  👋

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍

**Reply by cagix** [reply]
@jvanderhyde How did you manage to submit a JPEG? I'm offered only to take a live picture?

**Reply by jvanderhyde** [reply]
After all the problems I completely gave up on the idea of using GitHub for Education. ☹️ It was 5 months ago; I don't remember how I did it anymore. Sorry.

**Reply by cagix** [reply]
same here. thinking of switching platforms. sad.

**Reply by jvanderhyde** [reply]
I used GitHub anyway. I created an organization and invited all my students to join it. I put the code repos related to the course in the organization. It doesn't have any classroom features, but it still has all the GitHub features, which is what I wanted to teach them anyway.

**Reply by cagix** [reply]
just applied at gitlab.com and got verified without any problems. it seems like gitlab still cares for education, you get even answers from real people instead of talking to a dumb chatbot.

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent (relevant) activity and/or information may be outdated since the discussion was posted. Thank you for helping us maintain a productive and tidy community for all our members.

**Comment by ajayvardhanreddy-purdue** [comment]
Hello team,

I wanted to follow up regarding my application, which has been getting rejected even after I submitted all the required documents, including my class schedule, I-20, and admission letter. I am unsure if there might be an issue with the documents or if additional information is needed.

Could you please assist me in identifying and resolving the problem? Thank you for your help.

Best regards,
Ajay.

**Comment by marlin-2k** [comment]
can't upload because of ur retarded 1MB policy
